src/qt/transactiondesc.cpp

```diff
@@ -36,13 +36,27 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTxStatus& status
 {
     int depth = status.depth_in_main_chain;
     if (depth < 0) {
+        //: This status represents an unconfirmed transaction that conflicts with another.
```

Comment:
Suggesting to add more context on the location of this string and what it represents:

``` diff
diff --git a/src/qt/transactiondesc.cpp b/src/qt/transactiondesc.cpp
index c921b5fb1..0a52d2e3e 100644
--- a/src/qt/transactiondesc.cpp
+++ b/src/qt/transactiondesc.cpp
@@ -36,27 +36,41 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTxStatus& status
 {
     int depth = status.depth_in_main_chain;
     if (depth < 0) {
-        //: This status represents an unconfirmed transaction that conflicts with another.
+        /*: Text explaining the current status of a transaction, shown in the
+            status field of the details window for this transaction. This status
+            represents an unconfirmed transaction that conflicts with another
+            confirmed transaction. */
         return tr("conflicted with a transaction with %1 confirmations").arg(-depth);
     } else if (depth == 0) {
         QString s;
         if (inMempool) {
-            //: This status represents an unconfirmed transaction that is in the memory pool.
+            /*: Text explaining the current status of a transaction, shown in the
+                status field of the details window for this transaction. This status
+                represents an unconfirmed transaction that is in the memory pool. */
             s = tr("0/unconfirmed, in memory pool");
         } else {
-            //: This status represents an unconfirmed transaction that is not in the memory pool.
+            /*: Text explaining the current status of a transaction, shown in the
+                status field of the details window for this transaction. This status
+                represents an unconfirmed transaction that is not in the memory pool. */
             s = tr("0/unconfirmed, not in memory pool");
         }
         if (status.is_abandoned) {
-            //: This status represents an abandoned transaction.
+            /*: Text explaining the current status of a transaction, shown in the
+                status field of the details window for this transaction. This
+                status represents an abandoned transaction. */
             s += QLatin1String(", ") + tr("abandoned");
         }
         return s;
     } else if (depth < 6) {
-        //: This status represents a transaction confirmed in at least one block, but less than 6 blocks.
+        /*: Text explaining the current status of a transaction, shown in the
+            status field of the details window for this transaction. This
+            status represents a transaction confirmed in at least one block,
+            but less than 6 blocks. */
         return tr("%1/unconfirmed").arg(depth);
     } else {
-        //: This status represents a transaction confirmed in 6 or more blocks.
+        /*: Text explaining the current status of a transaction, shown in the
+            status field of the details window for this transaction. This status
+            represents a transaction confirmed in 6 or more blocks. */
         return tr("%1 confirmations").arg(depth);
     }
 }
```

Reply:
I have written about how translator comments should be written here: [translator-comments.md](https://github.com/bitcoin-core/gui-qml/blob/main/src/qml/doc/translator-comments.md)

Extra information on what it represents to the user and where it is helps the translator think of a correct translation but also allows them to locate the string themselves in the GUI, which would also aid with translations.

I also have a branch with these suggested changes here: https://github.com/jarolrod/gui/tree/tc-fixup

Reply:
Thanks. Great article. I updated the code with your suggestions.